Reverse-Proxy.md

```diff
@@ -121,12 +121,25 @@ Without SSL:
 ```
 
 # Caddy
-
+Install Caddy server
```

Comment:
What is happening right here?
I think you are trying to install caddy here => linking their installation guide is the appropriate way to do that

We cannot maintain a way to install caddy in the wiki.

```suggestion
After [instally caddy](https://caddyserver.com/docs/install), edit your `Caddyfile` (by default at `/etc/caddy/Caddyfile`) and add the following:
```